frontend/lib/about/dev_settings_view.dart

```diff
@@ -74,6 +77,10 @@ class DevSettingsView extends StatelessWidget {
             title: const Text('Set expired last card verification'),
             onTap: () => _setExpiredLastVerification(context),
           ),
+          ListTile(
+            title: const Text('Trigger self-verification'),
+            onTap: () => selfVerifyCard(userCodeModel, Configuration.of(context).projectId, client),
+          ),
```

Comment:
This was already implemented in the 'eine andere Karte verifizieren' screen. However, I also see no pain in having it in another place as well :)

Reply:
I think in the "eine andere Karte verifizieren", it does not trigger the same process. Here, it triggers the self-verification process which also persists the card verification